python/ray/actor.py

```diff
@@ -715,6 +716,21 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         placement_group_capture_child_tasks = actor_options[
             "placement_group_capture_child_tasks"
         ]
+        if placement_group != "default":
```

Comment:
Did we purge all uses of this in internal ray yet?

Reply:
Not yet, not sure if @rkooo567 has a plan to move the away from these three parameters with schedule_strategy.

Reply:
@jianoaix can you create an issue?